Controller/CRUDController.php

```diff
@@ -1087,7 +1079,7 @@ protected function handleModelManagerException(\Exception $e)
     }
 
     /**
-     * Redirect the user depend on this choice.
+     * Redirects the user depend on this choice.
```

Comment:
I still don't understand what this comment is supposed to mean.

Reply:
Just fixed the grammar. Any better description @greg0ire ?

Reply:
"redirects the user depending on his choice"? what does "this" refer to in your sentence?